React/Modules/RCTKeyboardObserver.m

```diff
@@ -94,6 +96,9 @@ - (void)EVENT:(NSNotification *)notification \
 
 static NSDictionary *RCTParseKeyboardNotification(NSNotification *notification)
 {
+#if TARGET_OS_TV
```

Comment:
I'd like to leave this as is for now and gut it later.  It's required by default in ScrollResponder.js.  Gutting it would require package manager changes for Apple TV so the JS layer can know when it's running on tvOS.